apps/prairielearn/src/ee/pages/institutionAdminGeneral/institutionAdminGeneral.ts

```diff
@@ -24,6 +25,10 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     const institution = await getInstitution(req.params.institution_id);
+    const authn_providers = (
+      await getInstitutionAuthenticationProviders(req.params.institution_id)
+    ).map((provider) => provider.name);
+    const availableTimezones = await getAvailableTimezonesByName();
```

Comment:
Nit: IMO it makes more sense to use `getAvailableTimezones()` here (that is, the function that returns `Timezone[]`).

Reply:
Updated this to `getAvailableTimezones() in [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6). Using `getAvailableTimezones()` makes this much easier to use!